trunner/test_runner.py

```diff
@@ -20,6 +24,69 @@ def resolve_project_path():
     return project_dir
 
 
+def init_logdir():
+    """Inits log directories if it's needed."""
+    if not trunner.ctx.logdir:
+        return
+
+    logdir = trunner.ctx.logdir
+    internal_logdir_names = ("out", "in", "inout")
+    if not os.path.isdir(logdir):
+        os.mkdir(logdir)
+
+    for dirname in internal_logdir_names:
+        if not os.path.isdir(f"{logdir}/{dirname}"):
+            os.mkdir(f"{logdir}/{dirname}")
+        # clear logfiles from the whole test campaign
+        else:
+            with open(f"{trunner.ctx.logdir}/{dirname}/test_campaign.log", "w"):
+                break
```

Comment:
I think you meant `pass` instead of `break`.

Reply:
yeah thanks, as the `with` solution is replaced it's outdated.